### DISCUSSION TITLE
CPU Intel 12900K not fully supported

### DISCUSSION THREAD

**Discussion by baekdoosixt** [mathoudebine/turing-smart-screen-python#56] [General]
Hello , first of all many thanks for the amazing work on this project.
As described in the object , i can't retreive the CPU temp on my brand new 12900k intel CPU.
But i'm able to get all the other info.
Do you have any clue on how i can solve this ?

Best Regards

**Comment by mathoudebine** [comment]
Hi, thanks for the feedback!
Are your on Windows?

Unfortunately on Windows the CPU temperature cannot be retrieved by `psutil` library used in this project: https://psutil.readthedocs.io/en/latest/index.html?highlight=temperature#sensors you can see that only Linux is supported for temperatures.
I will update [the wiki page for hardware support](https://github.com/mathoudebine/turing-smart-screen-python/wiki/System-monitor-:-hardware-support)

We will soon add support for LibreHardwareMonitor on this project, which will allow to get a lot of HW sensors directly from the same source on Windows. This will fix your issue.
Following feature request is open here: https://github.com/mathoudebine/turing-smart-screen-python/issues/52 you can subscribe to it to know the progress.

Stay tuned!

**Comment by baekdoosixt** [comment]
Thanks a lot for your quick reply !
I'll wait a for a future library update .
Keep up the good work !
